net-misc/ntp/ntp-4.2.8p11.recipe

```diff
@@ -0,0 +1,57 @@
+SUMMARY="A network protocol suite"
+DESCRIPTION="network time protocol suite/programs"
+HOMEPAGE="http://www.ntp.org"
+COPYRIGHT="1992-2010 University of Delaware"
+LICENSE="GNU GPL v2"
```

Comment:
These two definitely needs an update. See [Debian's `copyright`][0] for this package.

[0]: http://metadata.ftp-master.debian.org/changelogs/main/n/ntp/ntp_4.2.8p10+dfsg-6_copyright

Reply:
The one from Debian looks a bit off from the one from ntp: https://www.eecis.udel.edu/~mills/ntp/html/copyright.html

Reply:
Gentoo lincense references also other ones: http://gpo.zugaina.org/net-misc/ntp
Not sure which ones to use here :)

Reply:
I guess we have to extract them by hand then :) I'm a bit lazy rn for the task though

Reply:
Included the COPYRIGHT file created at build time from the accompanied html file as NTP license ...